TranscriberModels.py

```diff
@@ -11,17 +11,33 @@ def get_model(use_api):
 
 class WhisperTranscriber:
     def __init__(self):
-        self.audio_model = whisper.load_model(os.path.join(os.getcwd(), 'tiny.en.pt'))
+        self.lang = "en"
+        self.audio_model = self.load_model()
         print(f"[INFO] Whisper using GPU: " + str(torch.cuda.is_available()))
 
     def get_transcription(self, wav_file_path):
         try:
-            result = self.audio_model.transcribe(wav_file_path, fp16=torch.cuda.is_available())
+            result = self.audio_model.transcribe(wav_file_path, fp16=torch.cuda.is_available(), language = self.lang)
         except Exception as e:
             print(e)
             return ''
         return result['text'].strip()
     
+    def change_lang(self, language):
```

Comment:
```
def change_lang(self, language):
   self.lang = language
   load_model(language)
```

```
def load_model(language):
   if self.lang == "en":
       self.audio_model = whisper.load_model(os.path.join(os.getcwd(), 'tiny.en.pt'))
   else:
       self.audio_model = whisper.load_model(os.path.join(os.getcwd(), 'tiny.pt')) 
```

Reply:
I did know not to repeat myself but couldn't find this simple fix. Thank you!